staging/src/k8s.io/kubectl/pkg/cmd/autoscale/autoscale.go

```diff
@@ -189,7 +197,10 @@ func (o *AutoscaleOptions) Validate() error {
 	if o.Max < o.Min {
 		return fmt.Errorf("--max=MAXPODS must be larger or equal to --min=MINPODS, max: %d, min: %d", o.Max, o.Min)
 	}
-
+	// only one of the CPUPercent or CPU param is allowed
+	if o.CPUPercent != 0 && o.CPU != "" {
```

Comment:
The way you currently have this set is causing to fail when passing `--cpu` flag, because `--cpu-percent` default value is `-1`.

Reply:
```suggestion
	if o.CPUPercent < 0 && o.CPU != "" {
```
since in the description of the flag we clearly say that negative value is equal to not specified, and default autoscaling is being used.

Reply:
change to: 

```go
        // only one of the CPUPercent or CPU param is allowed
	if o.CPUPercent > 0 && o.CPU != "" {
		return fmt.Errorf("--cpu-percent and --cpu are mutually exclusive, CPUPercent: %v, CPU: %v", o.CPUPercent, o.CPU)
	}
```